_schema.graphql

```diff
@@ -11995,6 +11996,12 @@ type Query {
   # Find partners by IDs
   _unused_gravity_partners(ids: [ID!]!): [DoNotUseThisPartner!]
 
+  # Find a saved search by ID or criteria
+  _unused_gravity_savedSearch(
```

Comment:
question (non-blocking): For my own education, how does a field get prefixed with `_unused_gravity` and what does that mean?